src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -54,7 +54,7 @@ public function initRuntime(Environment $environment)
     {
         if ($this->renderer instanceof TwigRendererInterface) {
             $this->renderer->setEnvironment($environment);
-        } elseif (null !== $this->renderer) {
+        } elseif (is_array($this->renderer)) {
```

Comment:
This was not needed to fix the bug, right?

Reply:
It is needed:

```
Testing Symfony\Bridge\Twig\Tests\Extension\FormExtensionTest
.....E                                                              6 / 6 (100%)

Time: 132 ms, Memory: 4.00MB

There was 1 error:

1) Symfony\Bridge\Twig\Tests\Extension\FormExtensionTest::testAccessRendererAndInitRuntime with data set #2 (array(Mock_ContainerInterface_3ca94d5e Object (...), 'service_id'), Mock_FormRendererInterface_d5edefef Object (...))
Error: Cannot use object of type Mock_FormRendererInterface_d5edefef as array

/var/www/symfony/src/Symfony/Bridge/Twig/Extension/FormExtension.php:58
/var/www/symfony/src/Symfony/Bridge/Twig/Tests/Extension/FormExtensionTest.php:43

ERRORS!
Tests: 6, Assertions: 15, Errors: 1.

Reply:
Which is the problem reported here: https://github.com/sonata-project/SonataAdminBundle/issues/4652

Reply:
For anyone else wondering: The array stored in `$this->renderer` is replaced by the `FormRenderInterface` instance that is fetched from the container in `__get()`: https://github.com/dmaicher/symfony/blob/6813a9f1f1251412bf6161dd732e65e853680736/src/Symfony/Bridge/Twig/Extension/FormExtension.php#L124